Update indexer-batch.md (#8516)
magento/devdocs: src/guides/v2.4/extension-dev-guide/indexer-batch.md

</type>
```

### Setting batch size through environment variables.

> Starting from 2.4.2 its possbile to configure batch size through environmaent variables or in `app/etc/env.php` file for `cataloginventory_stock`, `catalog_category_product`, `catalogsearch_fulltext`, `catalog_product_price`, `catalogpermissions_category`, `inventory` indexers.

Reviewer: ```suggestion
Starting from 2.4.2 its possible to configure the batch size through the environment variables or in `app/etc/env.php` file for `cataloginventory_stock`, `catalog_category_product`, `catalogsearch_fulltext`, `catalog_product_price`, `catalogpermissions_category`, `inventory` indexers.
```

> Example of configuration in `app/etc/env.php`

Reviewer: ```suggestion
Example configuration in `app/etc/env.php`
```

> ```php

Reviewer: Need blank line before ```php and one after ```

<?php
return [
    'indexer' => [
        'batch_size' => [
            'cataloginventory_stock' => [
                'simple' => 200
            ],
            'catalog_category_product' => 666,
            'catalogsearch_fulltext' => 333,
            'catalog_product_price' => [
                'simple' => 200,
                'default' => 500,
                'configurable' => 666
            ],
            'catalogpermissions_category' => 999,
            'inventory' => [
                'simple' => 210,
                'default' => 510,
                'configurable' => 616
            ]
        ]
    ]
];
```

> Size for `catalog_category_product`, `catalogpermissions_category`, `catalogpermissions_category` batches can be setup for all product type only,

Reviewer: ```suggestion
Size for `catalog_category_product`, `catalogpermissions_category`, `catalogpermissions_category` batches can be setup for all product types only.
```

> Size for `cataloginventory_stock`, `catalog_product_price`, `inventory` can be set up for each product type. Also, if there is no batch size for specific product type, `default` value will be used. We suggest setting `default` value for each indexer that can have different batch sizes per product type.

Reviewer: ```suggestion
Size for `cataloginventory_stock`, `catalog_product_price`, `inventory` can be set up for each product type. Also, if there is no batch size for a specific product type, the `default` value will be used. It is recommended to set the `default` value for each indexer so that there can be different batch sizes per product type.
```
